entrypoint.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+cd /home/container
+
+# Information output
+echo "Running on Debian $(cat /etc/debian_version)"
+echo "Current timezone: $(cat /etc/timezone)"
+wine --version
+
+# Update Application
+if [ ! -z ${SRCDS_APPID} ] && [[ $AUTO_UPDATE == 1 ]]; then
+        if [ ! -z ${SRCDS_BETAID} ]; then
+                if [ ! -z ${SRCDS_BETAPASS} ]; then
+                        ./steamcmd/steamcmd.sh +login anonymous +force_install_dir /home/container +app_update ${SRCDS_APPID} -beta ${SRCDS_BETAID} -betapassword ${SRCDS_BETAPASS} +quit
+                else
+                        ./steamcmd/steamcmd.sh +login anonymous +force_install_dir /home/container +app_update ${SRCDS_APPID} -beta ${SRCDS_BETAID} +quit
+                fi
+        else
+                ./steamcmd/steamcmd.sh +login anonymous +force_install_dir /home/container +app_update ${SRCDS_APPID} +quit
+        fi
+fi
+
+if [[ $XVFB == 1 ]]; then
+        Xvfb :0 -screen 0 ${DISPLAY_WIDTH}x${DISPLAY_HEIGHT}x${DISPLAY_DEPTH} &
+fi
+
+# Install necessary to run packages
+echo "First launch will throw some errors. Ignore them"
+
+if [ ! -d $WINEPREFIX ]; then
+        mkdir $WINEPREFIX
+fi
+
+# Check if wine-gecko required and install it if so
+if [[ $WINETRICKS_RUN =~ gecko ]]; then
+        echo "Installing Gecko"
+        WINETRICKS_RUN=${WINETRICKS_RUN/gecko}
+
+        if [ ! -f "$WINEPREFIX/gecko_x86.msi" ]; then
+                wget -q -O $WINEPREFIX/gecko_x86.msi http://dl.winehq.org/wine/wine-gecko/2.47/wine_gecko-2.47-x86.msi
+        fi
+
+        if [ ! -f "$WINEPREFIX/gecko_x86_64.msi" ]; then
+                wget -q -O $WINEPREFIX/gecko_x86_64.msi http://dl.winehq.org/wine/wine-gecko/2.47/wine_gecko-2.47-x86_64.msi
+        fi
+
+        wine msiexec /i $WINEPREFIX/gecko_x86.msi /qn /quiet /norestart /log $WINEPREFIX/gecko_x86_install.log
+        wine msiexec /i $WINEPREFIX/gecko_x86_64.msi /qn /quiet /norestart /log $WINEPREFIX/gecko_x86_64_install.log
+fi
+
+# Check if wine-mono required and install it if so
+if [[ $WINETRICKS_RUN =~ mono ]]; then
+        echo "Installing mono"
+        WINETRICKS_RUN=${WINETRICKS_RUN/mono}
+
+        if [ ! -f "$WINEPREFIX/mono.msi" ]; then
+                wget -q -O $WINEPREFIX/mono.msi http://dl.winehq.org/wine/wine-mono/4.9.3/wine-mono-4.9.3.msi
+        fi
+
+        wine msiexec /i $WINEPREFIX/mono.msi /qn /quiet /norestart /log $WINEPREFIX/mono_install.log
+fi
+
+# List and install other packages
+for trick in $WINETRICKS_RUN; do
+        echo "Installing $trick"
+        winetricks -q $trick
+done
+
+# Replace Startup Variables
+MODIFIED_STARTUP=`eval echo $(echo ${STARTUP} | sed -e 's/{{/${/g' -e 's/}}/}/g')`
```

Comment:
Can you try to use the same startup as this one here - https://github.com/parkervcp/images/blob/bot/discordjs/entrypoint.sh#L8

Reply:
Sure